go/signedexchange/certurl/cert_chain.go

```diff
@@ -0,0 +1,56 @@
+package certurl
+
+import (
+	"crypto/x509"
+	"io"
+
+	"github.com/WICG/webpackage/go/signedexchange/cbor"
+)
+
+type CertChainItem struct {
+	Cert         *x509.Certificate // A parsed X.509 certificate.
+	OCSPResponse []byte            // DER-encoded OCSP response for Cert.
+	SCTList      []byte            // SignedCertificateTimestampList (Section 3.3 of RFC6962) for Cert.
+}
+
+type CertChain []CertChainItem
+
+// Write generates a certificate chain of application/cert-chain+cbor format and writes to w.
+// See https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#cert-chain-format for the spec.
+func (certChain CertChain) Write(w io.Writer) error {
+	enc := cbor.NewEncoder(w)
+
+	if err := enc.EncodeArrayHeader(len(certChain) + 1); err != nil {
+		return err
+	}
+	if err := enc.EncodeTextString("\U0001F4DC\u26D3"); err != nil {
```

Comment:
Would you name this const MagicString or something?

Reply:
Done.